src/bin/makemovie.py

```diff
@@ -1164,15 +1171,7 @@ def ProcessArguments(self):
         i = 0
         outputName = "movie"
         outputSpecified = 0
-        processingLA = 0
         while(i < len(commandLine)):
-            # If we're processing launch arguments for the parallel launcher
-            # then append the argument to the the list of launch args.
-            if processingLA == 1:
-                self.engineCommandLineProperties["launchArgs"] = \
```

Comment:
I think
```
self.engineCommandLineProperties["launchArgs"] += [commandLine[i]]
```
is clearer and works in python 2 and 3. Alternatively, maybe introducing a short variable...

```
eclp = self.engineCommandLineProperties
```

and using `eclp` everywhere simplifies even more.

Reply:
Why comment on removed code? I appreciate the info on the shortcut, but it seems to muddy the waters.

Reply:
> Why comment on removed code?

Doh!! Apologies. I wasn't paying attention